### DISCUSSION TITLE
Unable to download from france.tv

### DISCUSSION THREAD

**Discussion by Ryukugro** [aclap-dev/video-downloadhelper#532] [Help and Q&A]
### Your question

i wasn't able to download a program

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www.france.tv/slash/les-films-wakfu/5453787-oropo-bataille-pour-l-eliacube.html",
  "error": {
    "details": "SideDownload error: : [mp4 @ 000001d156bb1980] Could not find tag for codec none in stream #0, codec not currently supported in container\r\n[out#0/mp4 @ 000001d156be84c0] Could not write header (incorrect codec parameters ?): Invalid argument\r\nError opening output file C:\\Users\\Pierre\\Downloads\\Oropo Bataille pour lEliacube en replay - Les Films Wakfu.mp4.\r\nError opening output files: Invalid argument\r\n",
    "downloadable_id": "downloadable_7840682364242091",
    "error": "coapp_failure",
    "id": "coapp_failure_b2d8f1f2-d5c8-4320-94fb-789d679882cb"
  }
}

**Comment by GrampaWildWilly** [comment]
Je ne travaille pas pour VDH.  Je ne suis qu'un autre utilisateur, comme toi.

Dommage, mais le site impose du geoblock.  Il ne me permet pas d'accéder au site à cause de ma résidence aux États-Unis.  C'est possible que mes avis ici #530 soient utiles.  Possible.  Aucun garanti.  Mais possible.

**Comment by Mattmjs** [comment]
France.tv is a site using DRM, whether it's site wide or select content I don't know. I saw DRM on another example #435 
You can find out using Firefox like I show in the other discussion. It will not show in chrome or edge.
